source/release_notes/3.6.6-1.4.rst

```diff
@@ -0,0 +1,25 @@
+.. _3.6.6-1.4:
+
+=================
+|PSMDB| 3.6.6-1.4
+=================
+
+:Date: August 3, 2018
+:Download: http://www.percona.com/downloads/percona-server-mongodb-3.6/
+:Installation: :ref:`install`
+
+|PSMDB| is an enhanced, open source, and highly-scalable database that is
+a fully-compatible, drop-in replacement for MongoDB 3.6 Community Edition.
+It supports MongoDB 3.6 protocols and drivers.
+
+|PSMDB| extends Community Edition functionality by including the
+:ref:`inmemory` storage engine, as well as several enterprise-grade features.
+Also it includes :ref:`mongorocks` storage engine, which is now deprecated.
+|PSMDB| requires no changes to MongoDB applications or code.
+
+This release is based on `MongoDB 3.6.6 
+<https://docs.mongodb.com/manual/release-notes/3.6/#jul-10-2018>`_
+and provides the following additional changes:
+
+* :jirabug:`PSMDB-232`: |PSMDB| was crashing if got out of memory in case when
```

Comment:
this bug is not fixed yet.

Reply:
I have removed it as a blocker for the release currently, so I think it's safe to remove it from the release notes.